pootle/apps/accounts/utils.py

```diff
@@ -222,17 +224,17 @@ def revert_units_commented(self):
                 # translator_comment, commented_by, and commented_on
                 last_comment = comments.latest('pk')
                 unit.translator_comment = last_comment.new_value
-                unit.commented_by_id = last_comment.submitter_id
-                unit.commented_on = last_comment.creation_time
+                unit.change.commented_by_id = last_comment.submitter_id
+                unit.change.commented_on = last_comment.creation_time
```

Comment:
If you check a few lines above you are iterating changes, then you get the unit with `unit = change.unit` and now you call `unit.change` to set some fields on it. My question is if it is possible to just use `change` instead of `unit.change`.